samples/aspnetcore/blazor/FlightFinder/FlightFinder.Client/Components/SearchView.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@typeparam TSearchResult
+
+<div id="search-area" class="px-2 py-3">
+    @SearchContent
+</div>
+<div id="results-area">
+    <GreyOutZone IsGreyedOut=SearchInProgress>
+        <div class="px-4">
+            @if (Results != null)
+            {
+                <div class="title">
+                    <h2 class="my-3">@Results.Count results</h2>
+                    @if (Sort != null && Sort.Count > 0)
+                    {
+                        <select class="custom-select" bind=chosenSortOrder>
+                            @for (var i = 0; i < Sort.Count; i++)
+                            {
+                                <option value=@i>@Sort[i].DisplayName</option>
+                            }
+                        </select>
+                    }
+                </div>
+
+                foreach (var result in SortResults(Results))
+                {
+                    @SearchResultItemContent(result);
+                }
```

Comment:
The 'sort' widget could easily be its own control as well. I was considering making that part of the demo for this week - thoughts?

Reply:
If you think it fits well with the demo flow you have in mind, sounds good to me!